lib/script.js

```diff
@@ -18,14 +18,14 @@ if (window.history) {
     el.onclick = function (e) {
       e.preventDefault()
       var parser = document.createElement('a')
-      parser.href = el.href
-      var matchedTpl = _getTemplate(el.pathname)
+      parser.href = e.target.pathname
+      var matchedTpl = _getTemplate(e.target.pathname)
       if (matchedTpl) {
         document.write(matchedTpl)
         document.close()
-        history.pushState({}, '', el.href)
+        history.pushState({}, '', e.target)
       } else {
-        window.location = el.href
+        window.location = e.target
```

Comment:
So I made a test with a homepage linking 2 other pages (page2 and page 3)

with prod version of this plugin, i get :
```
e.target : http://liit.fr/page2
    parser.href : http://liit.fr/templates/page3
    matchedTpl : _getTemplate('/templates/page3')
    history.pushState : history.pushState({}, '', http://liit.fr/templates/page3)
    window.location : http://liit.fr/templates/page3
```
```
e.target : http://liit.fr/templates/page3
     parser.href : http://liit.fr/templates/page3
     matchedTpl : _getTemplate('/templates/page3')
     history.pushState : history.pushState({}, '', http://liit.fr/templates/page3)
     window.location : http://liit.fr/templates/page3
```
We can see it doesn't work with page2

With PR, we get : 
```
e.target : http://liit.fr/page2
     parser.href : http://liit.fr/page2
     matchedTpl : _getTemplate('/page2')
     history.pushState : history.pushState({}, '', http://liit.fr/page2)
     window.location : http://liit.fr/page2
```
```
e.target : http://liit.fr/templates/page3
     parser.href : http://liit.fr/templates/page3
     matchedTpl : _getTemplate('/templates/page3')
     history.pushState : history.pushState({}, '', http://liit.fr/templates/page3)
     window.location : http://liit.fr/templates/page3
```
looks good for me, urls are absolute

With your suggestion :
```
e.target : http://liit.fr/page2
     parser.href : http://liit.fr/page2
     matchedTpl : _getTemplate('/page2')
     history.pushState : history.pushState({}, '', http://liit.fr/page2)
     window.location : /page2
```
```
 e.target : http://liit.fr/templates/page3
    parser.href : http://liit.fr/templates/page3
    matchedTpl : _getTemplate('/templates/page3')
    history.pushState : history.pushState({}, '', http://liit.fr/templates/page3)
    window.location : /templates/page3
```
Here urls are relative but it looks like chrome of firefox don't care in navbar (they keep url absolute).

As you did in previous versions, I would keep urls absolute